src/y0/dsl.py

```diff
@@ -47,6 +48,9 @@ def to_latex(self) -> str:
     def __str__(self) -> str:
         return self.to_text()
 
+    def get_variables(self) -> Set[Variable]:
+        return {}
```

Comment:
`{}` will be an empty dict. You probably want `set()`

Reply:
well spotted. Corrected, also did a rebase on main.

Reply:
don't worry about rebasing. The PR will be squashed before merge!

Reply:
@maparent especially since some changes were lost during this rebase...

Reply:
which changes? the diff looks good to me. Did you make changes after my PR?

Reply:
Yes I did, I gave an explanation [above](https://github.com/y0-causal-inference/y0/pull/7#issuecomment-758674094). It's okay though since I had a copy of it locally. I added it again in https://github.com/y0-causal-inference/y0/pull/7/commits/507e8764e0feb80d75dc5d4bf3ad75833def9be3

Reply:
ah I see the changes now. Adding them back.

Reply:
ok at this point we're destroying one another's work. Do you mind letting me?

Reply:
Oh, dear. You merged my branch???

Reply:
wait, @maparent  it looks like the code returns a dictionary, but the signature specifies that it returns a set.  Should we chat quickly to synchronize?

Reply:
ok sorry for the last messages, my git state was quite confused. First time I experience someone else pushing to my PR. Made a few changes, also repaired the tests.